src/configurePipeline/services/target/azureTargetService.ts

```diff
@@ -0,0 +1,35 @@
+import { AzureTreeDataProvider } from "vscode-azureextensionui";
+
+import { AzureTargetResource, PipelineTargets } from "../../model/Common";
+import { WebAppTreeItem } from "../../tree/WebAppTreeItem";
+
+export class AzureTargetService {
+    private targetResource: AzureTargetResource;
+    private azureTreeDataProvider: AzureTreeDataProvider;
+
+    constructor(azureTreeDataProvider: AzureTreeDataProvider) {
+        this.azureTreeDataProvider = azureTreeDataProvider;
+    }
+
+    public async getTargetResource(): Promise<AzureTargetResource> {
+        let azureNode = <WebAppTreeItem>await this.azureTreeDataProvider.showTreeItemPicker("appService");
+        return this.extractTargetFromNode(azureNode);
+    }
+
+    public extractTargetFromNode(azureNode: WebAppTreeItem) {
+        this.targetResource = <AzureTargetResource>{
+            resourceId: azureNode.id,
+            resourceName: azureNode.label,
+            resourceType: PipelineTargets.WindowsWebApp,
```

Comment:
Just for my understanding, resource type will be derived here instead of constant right?

Reply:
This is no more in use